control/trajectory_follower_nodes/design/longitudinal_controller-design.md

```diff
@@ -142,10 +141,102 @@ Depending on the actuating principle of the vehicle, the mechanism that physical
+| drive_state_offset_stop_dist        | double | The state will transit to DRIVE when the distance to the goal is larger than `drive_state_stop_dist` + `drive_state_offset_stop_dist` [m]                     | 1.0           |
+| stopping_state_stop_dist            | double | The state will transit to STOPPING when the distance to the goal is smaller than `stopping_state_stop_dist` [m]                                               | 0.5           |
+| stopped_state_entry_vel             | double | ego velocity to be considered as STOPPED state [m/s]                                                                                                          | 0.01          |
+| stopped_state_entry_acc             | double | ego acceleration to be considered as STOPPED state [m/s^2]                                                                                                    | 0.1           |
+| emergency_state_overshoot_stop_dist | double | If `enable_overshoot_emergency` is tru and the ego is `emergency_state_overshoot_stop_dist`-meter ahead of the goal, the state will transit to EMERGENCY. [m] | 1.5           |
+| emergency_state_traj_trans_dev      | double | If the ego's position differs `emergency_state_traj_tran_dev` meter from the nearest trajectory point, the state will transit to EMERGENCY. [m]               | 3.0           |
+| emergency_state_traj_rot_dev        | double | If the ego's orientation differs `emergency_state_traj_rot_dev` rad from the nearest trajectory point, the state will transit to EMERGENCY. [rad]             | 0.784         |
+
+### DRIVE Parameter
+
+| Name                                  | Type   | Description                                                                                                                   | Default value |
+| :------------------------------------ | :----- | :---------------------------------------------------------------------------------------------------------------------------- | :------------ |
+| kp                                    | double | p gain for longitudinal control                                                                                               | 1.0           |
+| ki                                    | double | i gain for longidutinal control                                                                                               | 0.1           |
+| kd                                    | double | d gain for longidutinal control                                                                                               | 0.0           |
+| max_out                               | double | max value of output acceleration during DRIVE state [m/s^2]                                                                   | 1.0           |
```

Comment:
I think `max_out` is the max correction value for the original acceleration by PID.